search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -246,22 +241,21 @@ void shouldVerifySearchNotReturningHitsWithDraftPublicationRequestInSearchRespon
         var actualHitsExcludingHitsWithPublicationStatusDraft = 1;
         assertThat(searchResourcesResponse.getHits().size(),
                    is(equalTo(actualHitsExcludingHitsWithPublicationStatusDraft)));
-
-        indexingClient.deleteIndex(indexName);
     }
 
     @Test
     void shouldReturnPendingPublishingRequestsForDraftPublications()
         throws IOException, InterruptedException, BadGatewayException {
-        var indexName = generateIndexName();
 
         indexingClient.addDocumentToIndex(
             crateSampleIndexDocument(indexName, "sample_publishing_request_of_draft_publication.json"));
         indexingClient.addDocumentToIndex(
             crateSampleIndexDocument(indexName, "sample_publishing_request_of_published_publication.json"));
         Thread.sleep(DELAY_AFTER_INDEXING);
+
         var viewingScope = ViewingScope.create(ORGANIZATION_ID_URI_HARDCODED_IN_SAMPLE_FILES);
-        SearchTicketsQuery searchTicketsQuery = new SearchTicketsQuery(PAGE_SIZE, PAGE_NO);
+        var aggregations = ApplicationConstants.TICKETS_AGGREGATIONS;
+        SearchTicketsQuery searchTicketsQuery = new SearchTicketsQuery(PAGE_SIZE, PAGE_NO, aggregations);
```

Comment:
You should test the aggregation in a new test, rather then modifying a existing one

Reply:
Done.  Addtional:- Tested aggregation in SearchTicketsHandlerTest.